external-dependencies.md

```diff
@@ -0,0 +1,513 @@
+* **Language**: Go (Check go.mod)
+* **Dependencies:**
+  - github.com/gin-gonic/gin → MIT
+  - github.com/graphql-go/graphql → MIT
+  - github.com/sirupsen/logrus → MIT
+  - go.mongodb.org/mongo-driver → Apache-2.0
+Below is a ** table** listing the **BSS** repository’s external dependencies along with each module’s **version** and **commonly known license**. The table is split between **Direct** and **Additional (Indirect)** dependencies.
+> [!Note]
+> Always consult each dependency’s official repository and LICENSE file for the most accurate and current information. 
+
+### **Direct Dependencies**
+
+| **Dependency (Version)**                                                     | **License**                                                                                                                                              |
+|------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------|
+| github.com/Cray-HPE/hms-base (v1.15.1)                                       | Apache License 2.0                                                                                                                                       |
+| github.com/Cray-HPE/hms-certs (v1.4.0)                                       | Apache License 2.0                                                                                                                                       |
+| github.com/Cray-HPE/hms-compcredentials (v1.11.3)                            | Apache License 2.0                                                                                                                                       |
+| github.com/Cray-HPE/hms-go-http-lib (v1.5.4)                                 | Apache License 2.0                                                                                                                                       |
+| github.com/Cray-HPE/hms-securestorage (v1.13.0)                              | Apache License 2.0                                                                                                                                       |
+| github.com/DATA-DOG/go-sqlmock (v1.5.0)                                      | MIT                                                                                                                                                      |
+| github.com/Masterminds/squirrel (v1.5.4)                                     | MIT                                                                                                                                                      |
+| github.com/OpenCHAMI/jwtauth/v5 (v5.0.0-20240321222802-e6cb468a2a18)         | *(Not explicitly stated; presumed open-source. Possibly Apache 2.0 or MIT; verify in JWTAUTH repo.)*                                                      |
```

Comment:
This is our own build of https://github.com/go-chi/jwtauth/pull/93 with our own patch.